deployment/run.sh

```diff
@@ -210,16 +217,55 @@ es_ops_params=$(join , \
 	ES_RECOVER_AFTER_TIME=${es_ops_recover_after_time} \
 	)
 
-oc process -f templates/es.yaml -v "${es_params}" | oc create -f -
+if [[ -n "${ES_NODESELECTOR}" ]]; then
+	sed "/serviceAccountName/ i\
+\          $es_nodeselector" templates/es.yaml | oc process -v "${es_params}" -f - | oc create -f -
+else
+	oc process -f templates/es.yaml -v "${es_params}" | oc create -f -
+fi
+
 es_host=logging-es
 es_ops_host=${es_host}
-oc process -f templates/kibana.yaml -v "OAP_PUBLIC_MASTER_URL=${public_master_url},OAP_MASTER_URL=${master_url}" | oc create -f -
-oc process -f templates/curator.yaml -v "ES_HOST=${es_host},MASTER_URL=${master_url},CURATOR_DEPLOY_NAME=curator"| oc create -f -
+
+if [[ -n "${KIBANA_NODESELECTOR}" ]]; then
+	sed "/serviceAccountName/ i\
+\          ${kibana_nodeselecto}" templates/kibana.yaml | oc process -v "OAP_PUBLIC_MASTER_URL=${public_master_url},OAP_MASTER_URL=${master_url}" | oc create -f -
```

Comment:
suggest using `set -o nounset` to avoid these sorts of errors
it will be painful at first . . .